src/workerd/api/BUILD.bazel

```diff
@@ -92,16 +96,22 @@ wd_cc_library(
 
 wd_cc_library(
     name = "pyodide",
-    srcs = ["pyodide.c++"],
+    srcs = [
+        "pyodide.c++",
+        "pyodide/pyodide.c++",
```

Comment:
These should be merged together, or the second one given a more descriptive name.

Reply:
I'll leave that to a followup. But it would also be good to merge `pyodide.h` and `pyodide/pyodide.h`